ajax/libs/event-drops/package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "event-drops",
+	"npmName": "event-drops",
+	"npmFileMap": [
+		{
+		  "basePath": "src",
+		  "files": [
+	      "**/*"
+	    ]
+		}
+	],
+	"filename": "index.min.js",
```

Comment:
Should this be eventDrops.min.js instead ?

Reply:
There is no file named "eventDrops.js" in the repository hence I added index.js instead
Let me know if it's still wrong I will gladly make the changes